templates/profiles/membership.html

```diff
@@ -57,6 +57,12 @@ <h3>Medlemskap</h3>
                             studentweb.</p>
                         {{ field_of_study_application.documentation|as_crispy_field }}
                     </div>
+                    <div id="field-of-study-application-master-documentation">
+                        <p>For å søke master medlemsskap i linjeforeningen, kreves det at det gis et bevis på opptak.
+                            Dette må dokumenteres ved at du laster opp f.eks et screenshot av din studieplan på studentweb.
```

Comment:
If we're gonna be real Norwegians here we could name it "skjermdump", but that's up to you.

Anywho, "f.eks." needs a dot after "eks." too.

We could also be helpful and link to studentweb.ntnu.no.

While we're on it, we should probably coordinate with Hovedstyret regarding what's accepted as proof in case more options are available.